nwbwidgets/view.py

```diff
@@ -46,7 +46,9 @@ def show_dynamic_table(node, **kwargs) -> widgets.Widget:
     pynwb.behavior.Position: behavior.show_position,
     pynwb.behavior.SpatialSeries: OrderedDict({
         'over time': timeseries.SeparateTracesPlotlyWidget,
-        'trace': behavior.plotly_show_spatial_trace}),
+        'trace': behavior.plotly_show_spatial_trace,
+        'rate map': placefield.PlaceFieldWidget,
+        '1d rate map': placefield.PlaceField_1D_Widget}),
```

Comment:
Could we dynamically route this so that it uses 1D if the SpatialSeries is 1D?

Reply:
See https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets/blob/81b8b4f977dc7d40dc4b0f188d783b94f224a4fc/nwbwidgets/ophys.py#L245-L249